projects/applicationinsights-angularplugin-js/.eslintrc.json

```diff
@@ -16,6 +16,36 @@
         "createDefaultProgram": true
       },
       "rules": {
+        "indent": ["error"],                                        // Enabling for auto fixing
+        "quotes": ["warn", "double"],                               // Enabled for auto fixing
+        "curly": [ "error" ],                                       // Enabling for auto fixing
+        "brace-style": [ "error" ],                                 // Enabling for auto fixing
+        "prefer-const": "off",
+        "prefer-spread": "off",
+        "no-var": "off",
+        "no-extra-boolean-cast": "off",
+        "prefer-rest-params": "off",
+        "no-case-declarations": "off",
+        "no-prototype-builtins": "off",
+        "no-useless-escape": "off",                                 // Suppressing Error -- need to Review Later
+        "no-trailing-spaces": [ "warn", { "skipBlankLines": true }],// Enabled for auto fixing
+        "no-const-assign": "error",
+        "comma-dangle": [ "error", "never" ],                       // Enabled for auto fixing
+        "security/detect-object-injection": "off",                  // Suppress Warning -- need to Review Later
+        "security/detect-non-literal-fs-filename": "off",           // This was causing a bunch of false positive matches for dynamic config "watch" function
+        "@typescript-eslint/ban-types": "off",
+        "@typescript-eslint/no-unused-vars": [ "warn", { "vars": "all", "args": "none", "argsIgnorePattern": "^_", "ignoreRestSiblings": true } ],
+        "@typescript-eslint/triple-slash-reference": "off",
+        "@typescript-eslint/no-inferrable-types": "off",
+        "@typescript-eslint/no-this-alias": "off",
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-empty-function": "off",
+        "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "@typescript-eslint/no-extra-semi": "error",                // Enabled for auto fixing
+        "@typescript-eslint/no-non-null-assertion": "error",
+        "@typescript-eslint/dot-notation": "off",
+        "@typescript-eslint/quotes": "off",
```

Comment:
without adding this, would get "error Strings must use singlequote  @typescript-eslint/quotes"

Reply:
try setting to the same value as line 20